setup.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+ADAFRUIT_DIR=www/Adafruit_Python_GPIO
+
+if [ "${EUID}" -ne 0 ]; then
+	echo "Error: This script must be run as root."
+	exit 1
+fi
+
+if [ ! -e ${ADAFRUIT_DIR} ]; then
+	echo "Error: Adafruit directory not found."
+	exit 2
+fi
+
+apt-get install -y python python-dev python-pip python-smbus lighttpd
```

Comment:
Cool.
I think the commit of the lighttpd.conf makes sense to come with this pull request. [Here is mine for reference](https://gist.github.com/aninternetof/4432f1428fec367fe6ebd20207ef5766). I guess it won't need the php cgi stuff for much longer, but we can remove it later.